modules/op-release-notes-1-20-0.adoc

```diff
@@ -184,7 +184,7 @@ $ oc patch configmap pipelines-as-code -n openshift-pipelines --type=json -p='[{
 * Using `opc pr logs` in the OpenShift namespace may fail with repeated `Failed to list objects from openshift namespace` errors for both admin and non-admin users. 
 
 .{tekton-cache}
-* On IBM P environments, the `cache-fetch` step may fail with the error `failed to change ownership: operation not permitted`. This typically occurs because of filesystem permission restrictions on the underlying storage.
+* On IBM P and IBM Z environments, the `cache-fetch` step may fail with the error `failed to change ownership: operation not permitted`. This typically occurs because of filesystem permission restrictions on the underlying storage.
```

Comment:
Suggestion: On IBM P and IBM Z environments, the `cache-fetch` step might fail with the `failed to change ownership: operation not permitted` error message. This issue occurs due to filesystem permission restrictions on the underlying storage.